<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startNowbtn">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>598</y>
      <width>437</width>
      <height>153</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Start now</string>
    </property>
   </widget>
   <widget class="QLabel" name="signInbtn">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>779</y>
      <width>154</width>
      <height>48</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Or Sign in</string>
    </property>
   </widget>
   <widget class="QLabel" name="heading">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>181</y>
      <width>771</width>
      <height>282</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>50</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Lorem ipsum dolor sit 
 amet consectetur 
 adipiscing?</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
